mobile/test/common/integration/base_client_integration_test.cc

```diff
@@ -194,9 +194,8 @@ void BaseClientIntegrationTest::createEnvoy() {
     }
   }
 
-  finalizeConfigWithPorts(config_helper_, ports, use_lds_);
-
   if (override_builder_config_) {
```

Comment:
`override_builder_config_` is always `false` now, so this is dead code.  But I remember Alyssa mentioned it might be good to leave it in in case we needed to use `override_builder_config_` again?